src/routes/movies.routes.ts

```diff
@@ -200,15 +218,21 @@ router.put('/:id', (req: Request, res: Response, next: NextFunction) => {
  *       500:
  *         description: Internal Server Error
  */
-router.delete('/:id', (req: Request, res: Response, next: NextFunction) => {
-  const movieId = req.params.id;
+router.delete(
+  '/:id',
+  async (req: Request, res: Response, next: NextFunction) => {
+    try {
+      const movieId = req.params.id;
+
+      const deletedMovie: Movie | null =
+        await MovieModel.findByIdAndDelete(movieId);
```

Comment:
The same situation as in deleting genres

Reply:
added check for null